### PR TITLE
Update readiness-assessment-fix.md

### PR DESCRIPTION
**Pull request by ElaineAnsell**
Revised description and Not ready and Advisory text for Conditional Access and MFA to be consistent with the recent changes we made in the Adjust settings doc article for the same two settings

@jaimeo , these edits reflect the changes we made two weeks ago with Adam's input to the Adjust settings doc. Thank you!

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by ElaineAnsell** [comment]
Adding @Jaime Ondrusek<mailto:jaimeo@microsoft.com> for additional review as our technical writer as this related to Microsoft Managed Desktop. Thanks!

Regards,
Elaine

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 1]
```diff
@@ -145,16 +145,16 @@ Microsoft Store for Business either isn't enabled or isn't synced with Intune. F
...
+You have some multifactor authentication policies set as **required** for conditional access policies that are assigned to all users. At Microsoft Managed Desktop enrollment, Microsoft Managed Desktop will modify conditional access policies that prevent Microsoft Managed Desktop from managing the Microsoft Managed Desktop service.  During enrollment, Microsoft Managed Desktop will exclude Microsoft Managed Desktop service accounts from relevant conditional access policies and apply Microsoft Managed Desktop managed conditional access policies to restrict access to these accounts. For more about these service accounts, see [Standard operating procedures](../service-description/operations-and-monitoring.md#standard-operating-procedures).
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]:
```suggestion
You have some multifactor authentication policies set as **required** for conditional access policies that are assigned to all users. At Microsoft Managed Desktop enrollment, Microsoft Managed Desktop will modify conditional access policies that prevent Microsoft Managed Desktop from managing the Microsoft Managed Desktop service. During enrollment, Microsoft Managed Desktop will exclude Microsoft Managed Desktop service accounts from relevant conditional access policies and apply Microsoft Managed Desktop managed conditional access policies to restrict access to these accounts. For more about these service accounts, see [Standard operating procedures](../service-description/operations-and-monitoring.md#standard-operating-procedures).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@ElaineAnsell

**Comment by jaimeo** [comment]
This needs some additional edits for concision and clarity. I'm working with @ElaineAnsell separately and will edit here directly.

**Comment by jaimeo** [comment]
Note to copy editors: we avoid "may." Use "might" or "could."

**Pull request merged**